### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by 29venom1**
check this solution

### CODE REVIEW

**Comment by zre12f** [comment]
it sims right

**Comment by 29venom1** [comment]
> it sims right

R u sure?

**Comment by zre12f** [comment]
> > it sims right
> 
> R u sure?

yep

File: 6_kyu/Mexican Wave.py [hunk 1]
```diff
@@ -1,3 +1,7 @@
 def wave(people):
-    # Code here
-    pass
\ No newline at end of file
+    print([people[:c]+people[c].upper()+people[c+1:] for c in range(len(people))if people[c] != ' '])
```
Comment (zre12f) [inline 6_kyu/Mexican Wave.py:2]: maybe you'd use the 'return' command?

Comment (29venom1) [reply 6_kyu/Mexican Wave.py:2]: > maybe you'd use the 'return' command?

yes. u r right!

**Pull request merged**